### ISSUE TITLE
Quel hyperparamètres optimiser pour le modèle GAM ?

### ISSUE THREAD

**Issue by vviers** [signaux-faibles/rsignauxfaibles#59]
Dans le fichier `R/model_optimize.R`, la fonction `optimize_hyperparameters` s'applique à des paramètres de l'ancien modèle (xgboost)

Elle est donc aujourd'hui cassée. Ce qui pose la question des hyperparamètres à tuner dans le modèle GAM. 

J'en vois pour ma part assez peu :
- quelles fonctions de base (basis functions) --> par défaut, des splines ?
- `k` : nombre de noeuds dans les fonctions de base / combien de combien de degrés de liberté effectif (effective degrees of freedom / EDF) --> les défauts de mcvg semblent sensibles

J'ai envie de dire que pour l'instant, `optimize_hyperparameters`  ne sert a rien dans le contexte du nouveau modèle.

WDYT ?

**Comment by arianelestrade** [comment]
Je suis d'accord, pour l'instant cette fonction ne sert à rien dans ce contexte. Je vais me renseigner sur les paramètres à optimiser en priorité, et je te dis si je vois d'autres choses. 
Honnêtement, plus on avance sur le sujet de la modélisation, plus je suis partante d'avoir d'abord une structure stable où on puisse lancer un modèle et l'évaluer correctement, en privilégiant des modèles simples qu'on connait bien et faciles à implémenter (par exemple régression logistique, régression LASSO, forêt aléatoire, et éventuellement GAM avec des paramètres un peu par défaut). Pour moi l'important c'est d'avoir un modèle qui tourne, qui s'évalue, qui s'appuie sur le maximum de données à notre disposition (notamment celles d'Altharès!), et de voir la correction adhoc par secteur !
